extensions/default/src/utils/reuseCachedLayouts.ts

```diff
@@ -0,0 +1,75 @@
+import { HangingProtocolService, StateSyncService, Types } from '@ohif/core';
+
+export type ReturnType = {
+  hangingProtocolStageIndexMap: Record<string, Types.HangingProtocol.HPInfo>;
+  viewportGridStore: Record<string, unknown>;
+  displaySetSelectorMap: Record<string, string>;
+};
+
+/**
+ * Calculates a set of state information for hanging protocols and viewport grid
+ * which defines the currently applied hanging protocol state.
+ * @param state is the viewport grid state
+ * @param syncService is the state sync service to use for getting existing state
+ * @returns Set of states that can be applied to the state sync to remember
+ *   the current view state.
+ */
+const reuseCachedLayout = (
```

Comment:
Is this even used?

Reply:
Yes, it is used in the commandsModule.  Quite important as it is the core of the remembering when changing viewports.

Reply:
Ah ok. I see that CommandsModule imports it as `reuseCachedLayouts` (i.e. it adds an s to it).